python/requirements.txt

```diff
@@ -1,6 +1,32 @@
-pip==9.0.3
-setuptools==44.0.0
-wheel==0.30.0a0
+#
```

Comment:
We'd be able to get rid of this requirements file once `piptool` is fully removed from the rules. Would we then remove the [macro usage here](https://github.com/bazelbuild/rules_python/pull/373/files#diff-17b05993903e40e70b6caf9066a8a292149f43c5ba7f9f79373e8f29b61a70c6R135), or is there a reason to keep it?

Reply:
sure if the requirements file goes away, then so should the rule that keeps it updated.